src/manifest.ts

```diff
@@ -28,12 +39,33 @@ interface ReleaserConfig {
 
 interface ReleaserPackageConfig extends ReleaserConfig {
   'package-name'?: string;
+  'changelog-path'?: string;
 }
 
 export interface ManifestConfig extends ReleaserConfig {
   packages: Record<string, ReleaserPackageConfig>;
 }
 
+interface Package {
+  path: string;
+  releaseType: ReleaseType;
+  packageName?: string;
+  bumpMinorPreMajor?: boolean;
+  changelogSections?: ChangelogSection[];
+  changelogPath?: string;
```

Comment:
Are these fields included in the manifest? I still feel the releaser configuration should not go into the manifest. IMO, the manifest should include the package name, last release version, and last release SHA.

The GitHub app config, CLI script, or GitHub action should have the releaser configuration and doesn't need to be duplicated in the manifest.

Reply:
100% agree with you. These fields are not in the manifest (see [this PR's description for details](https://github.com/googleapis/release-please/pull/785#issue-576524915) - the manifest file format is easy to miss because it's only a single line near the top of the description).

`Package` here is an internal interface representing the parsing that happens on the releaser config - a dumbed down version of what yargs does for our cli flags.